ansible/group_vars/all.yml

```diff
@@ -116,7 +117,7 @@ broker_bootstrap_refresh_interval: 600s
 broker_sandbox_role: edit
 
 broker_image_name: docker.io/ansibleplaybookbundle/origin-ansible-service-broker
-broker_tag: "latest"
+broker_tag: "nightly"
```

Comment:
Any reason this is being changed to nightly?  
I think we should default to "latest"